### DISCUSSION TITLE
Added: LDM notes for January 24th, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#5679] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-01-24.md

## Agenda

1. Required members metadata representation
2. Default implementations of abstract statics
3. Triage
    1. Nested members in with and object creation
    2. Binary Compat Only
    3. Attribute for passing caller identity implicitly
    4. Attributes on Main for top level programs

**Comment by MgSam** [comment]
Probably should have asked this closer to the release of C# 10, but has the LDT triaged the highest voted items in Discussions at any point?  The public planning board you guys used to have under Projects disappeared some time ago. 

Seems like user popularity should have some weight in planning future versions.

**Reply by CyrusNajmabadi** [reply]
Yes. We continually triage. We will likely be doing so today as well

**Comment by sab39** [comment]
"My new goal is to be able to say C# 11 has a 'spaceship operator'. I don't care what it does."

How about
```cs
public int Foo <=> impl.Foo;
```
as shorthand for
```cs
public int Foo { get => impl.Foo; set => impl.Foo = value; }
```
